backend/index.ts

```diff
@@ -24,35 +29,29 @@ import { getAccountFromAuth } from "./services/auth.js";
   return this.toString();
 };
 
-const app = Express();
-app.use(Express.json());
+const app = express();
+app.use(express.json());
 
 // TODO: See if we want to generate the OpenAPI doc instead of spec first
 const api = new openapiBackend.OpenAPIBackend({
   definition: "openapi.json",
   handlers: {
+    ...assets,
     ...auth,
     ...content,
     ...graph,
     ...profile,
 
-    validationFail: async (c, req: Express.Request, res: Express.Response) =>
+    validationFail: async (c, req: Request, res: Response) =>
       res.status(400).json({ err: c.validation.errors }),
-    notFound: async (c, req: Express.Request, res: Express.Response) =>
+    notFound: async (c, req: Request, res: Response) =>
       res.status(404).json({ err: "not found" }),
   },
 });
 
-api.init();
-
-// cors
-app.use(cors());
-
-// logging
-app.use(morgan("combined"));
-
-// Swagger UI
-app.use("/docs", swaggerUi.serve, swaggerUi.setup(openapiJson));
+api.register("unauthorizedHandler", (_c, _req, res) => {
+  return res.status(401).send();
```

Comment:
Do we want to send an error message?

Reply:
You're right. Adding a message is a good idea. However, I'm not sure if a message is already added from the open-api-backend spec. Since I didn't add this code I'll create a new issue to look into it and add the message if needed.